src/fides/lib/models/client.py

```diff
@@ -40,6 +42,7 @@ def __tablename__(self) -> str:
     salt = Column(String, nullable=False)
     scopes = Column(ARRAY(String), nullable=False, server_default="{}", default=dict)
     roles = Column(ARRAY(String), nullable=False, server_default="{}", default=dict)
+    systems = Column(ARRAY(String), nullable=False, server_default="{}", default=dict)
```

Comment:
When adding a user to a system or removing a user from the system, their client.systems array will be updated with that system's id so it's ready to persist to the token on login.